docs/usage.rst

```diff
@@ -1,156 +1,102 @@
+* May need to deactivate environment and reactivate it in order for changes to apply
+* ``fre/setup.py`` allows ``fre/fre.py`` to be ran as ``fre`` on the command line by defining it as an
+  *entry point*. Without it, the call would be instead, something like ``python fre/fre.py``
 
 
 Tools
------
+=====
 
 A few subtools are currently in development:
 
-**fre pp**
 
-1. configure 
+fre app
+-------
 
-* Postprocessing yaml configuration
-* Minimal Syntax: *fre pp configure -y [user-edit yaml file]*
-* Module(s) needed: n/a
-* Example: *fre pp configure -y /home/$user/pp/ue2/user-edits/edits.yaml*
-
-2. checkout
-
-* Checkout template file and clone gitlab.gfdl.noaa.gov/fre2/workflows/postprocessing.git repository
-* Minimal Syntax: *fre pp checkout -e [experiment name] -p [platform name] -t [target name]*
-* Module(s) needed: n/a
-* Example: *fre pp checkout -e c96L65_am5f4b4r0_amip -p gfdl.ncrc5-deploy -t prod-openmp*
+1. ``generate-time-averages``
+2. ``regrid_xy``
 
+   
+fre catalog
+-----------
 
-**fre catalog**
+1. ``builder`` Generate a catalog
 
-1. buildCatalog1
 * Builds json and csv format catalogs from user input directory path
-* Minimal Syntax: *fre catalog buildCatalog -i [input path] -o [output path]*
+* Minimal Syntax: ``fre catalog builder -i [input path] -o [output path]``
 * Module(s) needed: n/a
-* Example: *fre catalog buildCatalog -i /archive/am5/am5/am5f3b1r0/c96L65_am5f3b1r0_pdclim1850F/gfdl.ncrc5-deploy-prod-openmp/pp -o ~/output --overwrite*
-
-**To be developed:**
-
-#. fre check
-#. fre list
-#. fre make
-#. fre run
-#. fre test
-#. fre yamltools
-
-
-Usage (Developers)
-------------------
-
-Developers are free to use the user guide above to familiarize with the CLI and save time from having to install any dependencies, but development within a Conda environment is heavily recommended regardless
-
-Gain access to the repository with *git clone --recursive git@github.com:NOAA-GFDL/fre-cli.git* or your fork's link (recommended) and an SSH RSA key
-
-Once inside the repository, developers can test local changes by running a *pip install .* inside of the root directory to install the fre-cli package locally with the newest local changes
-
-Test as a normal user would use the CLI
+* Example: ``fre catalog builder -i /archive/am5/am5/am5f3b1r0/c96L65_am5f3b1r0_pdclim1850F/gfdl.ncrc5-deploy-prod-openmp/pp -o ~/output --overwrite``
 
-**Adding New Tools - Checklist**
+2. ``validate`` Validate the catalog
 
-If there is *no* subdirectory created for the new tool you are trying to develop, there are a few steps to follow:
 
-1. Create a subdirectory for the tool group inside the /fre folder; i.e. /fre/fre(subTool)
+fre cmor
+--------
 
-2. Add an *__init__.py* inside of the new subdirectory
+1. ``run``
 
-* This will contain one line, *from fre.fre(subTool) import **
-* The purpose of this line is to allow the subTool module to include all the scripts and functions within it when invoked by fre
+* placehold
```

Comment:
not sure if this placeholder is intended to be here, there's a coupler more on this file.

Reply:
i have https://github.com/NOAA-GFDL/fre-cli/pull/255 to fill in the `fre cmor` subsection next. other tools have placeholders for now so others easily know where their info is wanted!